src/engine/ruleset/environments/wazuh-environment.yml

```diff
@@ -61,5 +61,11 @@ decoders:
   # Service decoders
   - decoder/syslog/0
 
+  # Fortinet
```

Comment:
Improve-desing: Guiding me from the other decoders and the last redis merged decoder, I would say that this should be put together with the other decoders and they should be put in alphabetical order

Reply:
Done!